src/frontend/tsx/GamePanel.tsx

```diff
@@ -194,7 +199,7 @@ export function GamePanel () {
             Accept Undo Request</button>
           : <button
             className='option'
-            disabled={!hasGame || displayGame.sentUndoRequest}
+            disabled={!hasGame || (displayGame as ActiveGameInfo).sentUndoRequest}
             onClick={requestUndoOnClick}>
             Request to Undo Move</button>
         }
```

Comment:
Conditionals are messed up for the (reset) practice board button (might have been me by accident during rebase).
- line `210` can be removed
- line `215` should be just `disabled={!practiceHadMoved}`